### PR TITLE
bug 1193448: separate nova_host and vncproxy_host

### PR DESCRIPTION
**Pull request by sseago**
Change nova_host to use nova_private_vip, and add
vncproxy_host to stick with nova_public_vip

### CODE REVIEW

**Comment by sseago** [comment]
This depends on OFI PR https://github.com/redhat-openstack/astapor/pull/491

File: app/lib/staypuft/seeder.rb [hunk 1]
```diff
@@ -386,14 +386,15 @@ def functional_dependencies
...
+      amqp_host     = get_host_format :amqp_vip
+      mysql_host    = get_host_format :db_vip
+      glance_host   = get_host_format :glance_private_vip
+      neutron_host  = get_host_format :neutron_private_vip
+      nova_host     = get_host_format :nova_private_vip
       #admin API
-      auth_host    = get_host_format :keystone_admin_vip, Staypuft::SubnetType::ADMIN_API
+      auth_host     = get_host_format :keystone_admin_vip
       # public API
-      nova_host    = get_host_format :nova_public_vip, Staypuft::SubnetType::PUBLIC_API
```
Comment (bcrochet) [inline app/lib/staypuft/seeder.rb]: Don't you still need this line?

Comment (sseago) [inline app/lib/staypuft/seeder.rb]: It's moved from public API to private API network -- see line 393.

Comment (bcrochet) [inline app/lib/staypuft/seeder.rb]: I see it... sorry... forest and trees.

**Comment by bcrochet** [comment]
LGTM

**Pull request merged**